notebooks/algorithm-design/applications.ipynb

```diff
@@ -0,0 +1,1720 @@
+    "\n",
+    "In this case we will use, Terra's [`Estimator`](https://qiskit.org/documentation/stubs/qiskit.primitives.Estimator.html#qiskit.primitives.Estimator) and the optimizer will be [`SLSQP`](https://qiskit.org/documentation/stubs/qiskit.algorithms.optimizers.SLSQP.html) (i.e. Sequential Least SQuares Programming). Additionally, we will set the initial point for the `SLSQP` optimizer to $\\vec\\theta_0 = (1, \\cdots, 1)$."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from qiskit.primitives import Estimator\n",
+    "from qiskit.algorithms.optimizers import SLSQP\n",
+    "from qiskit.algorithms.minimum_eigensolvers import VQE\n",
+    "import numpy as np\n",
+    "\n",
+    "estimator = Estimator()\n",
+    "optimizer = SLSQP()\n",
+    "vqe = VQE(estimator, ansatz, optimizer, initial_point=np.ones(8))"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now that we have initialized the `VQE` instance, we can get the results with the [`VQE.compute_minimum_eigenvalue`](https://qiskit.org/documentation/stubs/qiskit.algorithms.minimum_eigensolvers.VQE.compute_minimum_eigenvalue.html#qiskit.algorithms.minimum_eigensolvers.VQE.compute_minimum_eigenvalue) method. Let us look at the results."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "result = vqe.compute_minimum_eigenvalue(observable_1)\n",
+    "print(result)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "The [`VQEResult`](https://qiskit.org/documentation/stubs/qiskit.algorithms.minimum_eigensolvers.VQEResult.html) we obtain explains the following:\n",
+    "* `aux_operators_evaluated`: the expected values of any auxiliar operators we might include.\n",
+    "* `cost_function_evals`: Number of cost function evaluations.\n",
+    "* `eigenvalue`: the resulting eigenvalue.\n",
+    "* `optimal_circuit`: the parametrized circuit from which the solution was found. To find the eigenstates you have to bind the optimal values to the parameters.\n",
+    "* `optimal_parameters`: The set of optimal parameters as a dictionary\n",
+    "* `optimal_point`: an array with the optimal parameter values\n",
+    "* `optimal_value`: the optimal value of the cost function.\n",
+    "* `optimizer_evals`: number of optimizer evaluations.\n",
+    "* `optimizer_result`: results of the optimizer as an [`OptimizerResult`](https://qiskit.org/documentation/stubs/qiskit.algorithms.optimizers.OptimizerResult.html) object.\n",
+    "* `optimizer_time`: time used for optimization (in seconds)."
```

Comment:
I'd just mention a couple of useful ones here (probably just `optimizer_result` and `optimal_point`) and link out to the API docs.